cdc/sorter/sorter.go

```diff
@@ -24,5 +24,7 @@ import (
 type EventSorter interface {
 	Run(ctx context.Context) error
 	AddEntry(ctx context.Context, entry *model.PolymorphicEvent)
+	// TryAddEntry try to add entry to the sorter, returns false if the entry can not be added, otherwise returns true
+	TryAddEntry(ctx context.Context, entry *model.PolymorphicEvent) bool
```

Comment:
Do we need it to return an error? It may be useful to know that the sorter has been cancelled or otherwise irrecoverably unavailable.